gwardia/templates/gwardia/index.html

```diff
@@ -3,5 +3,35 @@
 {% block title %}Panel Gwardii{% endblock %}
 
 {% block content %}
-    <h1>Panel Gwardii! WIP!</h1>
+    <div class="flex flex-col items-center">
+        <h1 class="text-4xl text-white font-bold mt-16 mb-8">Panel Gwardii! WIP!</h1>
+        <p class="italic text-xs text-gray-500 mb-2">Kliknij w dane spotkanie, by wyświetlić jego opis</p>
+        <ul class="w-1/2" id="js-next-meetings-container">
+            {% for meeting in upcoming_five_meetings %}
+                <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer">
+                    <p class="w-full">
+                        <span class="break-all">{{ meeting.title }}</span><span class="float-right">{{ meeting.date_of_meeting }}</span>
```

Comment:
Why in one line?

Reply:
It's short, so

Reply:
123 characters (143 with indent). Too big!

Reply:
okok